lib/fs.js

```diff
@@ -735,6 +735,12 @@ fs.write = function(fd, buffer, offset, length, position, callback) {
     if (typeof position === 'function') {
       position = null;
     }
+    if (typeof offset !== 'number') {
+      offset = 0;
+    }
+    if (typeof length !== 'number') {
+      length = buffer.length - offset;
+    }
```

Comment:
The comment at the top of the function does not match public documentation. Mind fixing that.